src/components/Garden/ActionCollateral.js

```diff
@@ -15,12 +15,12 @@ function ActionCollateral({ proposal }) {
       `}
     >
       <img
-        src={honeyIconSvg}
+        src={getTokenIconBySymbol(collateralRequirement.tokenSymbol)} // TODO: Use deefault-token-list
```

Comment:
I think that's fine though since it would just show the default token icon. I don't have strong feelings with keeping it this way or using the `default-token-list` but i think thos way we don't maintain duplicated metadata.

Nevertheless it should be noted that if we go with our current implementation we will need to update this function (`getTokenIconBySymbol `), since it just returns the icon from a mapping of harcoded symbol to icon, it's not a general solution.

Reply:
> maintain duplicated

yeah in any case we can fetch it from the default token list and fallback to the dao list in case not found it